Trying to load weight dict from /home/jgacon/uni/brexit-project/data/weight.json..
Traceback (most recent call last):
  File "run.py", line 24, in <module>
    test2()
  File "run.py", line 19, in test2
    l = Loader(data_dir + "brexit_data.json")
  File "/home/jgacon/uni/brexit-project/loader/loader.py", line 12, in __init__
    self.load_file(filename)
  File "/home/jgacon/uni/brexit-project/loader/loader.py", line 24, in load_file
    self.data = pandas.read_json(io)
  File "/usr/lib64/python2.7/site-packages/pandas/io/json.py", line 234, in read_json
    date_unit).parse()
  File "/usr/lib64/python2.7/site-packages/pandas/io/json.py", line 302, in parse
    self._parse_no_numpy()
  File "/usr/lib64/python2.7/site-packages/pandas/io/json.py", line 519, in _parse_no_numpy
    loads(json, precise_float=self.precise_float), dtype=None)
ValueError: Expected object or value
